source/monitoring.txt

```diff
@@ -0,0 +1,383 @@
+==================
+Cluster Monitoring
```

Comment:
S: this page covers cluster, command, and connection pool monitoring, so I'd rename to just "monitoring"
```suggestion
Monitoring
```